### PR TITLE
fix domain on pv with wdt sample

### PR DESCRIPTION
**Pull request by bhavaniravichandran**
When updating the domain-on-pv using wdt sample, DynamicClusterSize was mistakenly set to DynamicClusterSize to initialReplicaCountServers. This caused only 2 servers to be configured. But setting it to ConfiguredManagedServerCount, all the servers were started. The fix is to set DynamicClusterSize to ConfiguredManagedServerCount in the topology section of the model file. Also, set replicas to 2 and added a cluster section with the initial cluster (cluster-1) and its replicas in the kubernetes section. This started the replica count of cluster-1 servers and when a second cluster was added, started 2 server in the second cluster. I also verified scaling up and down both the clusters and they work as expected.
Also fixed the issue - weblogic-deploy.zip fails to download because the proxy is not set int he job pod that creates the domain

### CODE REVIEW

**Comment by bhavaniravichandran** [comment]
Started jenkins job and I will update here when done

File: kubernetes/samples/scripts/common/utility.sh [hunk 1]
```diff
@@ -102,6 +102,12 @@ function parseCommonInputs {
     fail 'The file ${exportValuesFile} could not be found.'
   fi
 
+  # Pass the https_proxy value to the job pod. We might need it to download WDT zip file
+  echo https_proxy = ${https_proxy}
+  proxyStr="https_proxy=${https_proxy}"
+  echo proxyStr=${proxyStr}
```
Comment (anpanigr) [inline kubernetes/samples/scripts/common/utility.sh:108]: remove  extraneous echo  statement  "echo https_proxy = ${https_proxy}"

File: kubernetes/samples/scripts/common/utility.sh [hunk 2]
```diff
@@ -474,6 +482,8 @@ function createFiles {
     # Generate the yaml to create the kubernetes job that will create the weblogic domain
     echo Generating ${createJobOutput}
 
+    echo httpsProxy=${httpsProxy}
```
Comment (anpanigr) [inline kubernetes/samples/scripts/common/utility.sh:485]: Extra echo statements

File: kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/wdt/create-domain-script.sh [hunk 3]
```diff
@@ -100,22 +100,22 @@ function install_wdt {
...
-    sleep 10
-    count=`expr $count + 1`
-    for proxy in "${https_proxy}" "${https_proxy2}"; do
-	  echo @@ "Info:  Downloading $WDT_INSTALL_ZIP_URL with https_proxy=\"$proxy\""
-	  https_proxy="${proxy}" \
-	    curl --silent --show-error --connect-timeout 10 -O -L $WDT_INSTALL_ZIP_URL 
-	  curl_res=$?
-	  [ $curl_res -eq 0 ] && break
-	done
+  #max=20
```
Comment (anpanigr) [inline kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/wdt/create-domain-script.sh:103]: Remove commented code

**Review by anpanigr — COMMENTED** [review]
Remove extraneous echo statements and commented code in the script.

**Review by anpanigr — COMMENTED** [review]
Remove  Statement "echo useWdt is ${useWdt}" 
from samples/scripts/common/validate.sh and samples/scripts/common/utility.sh

Remove statement  " echo Printing domainHomeSourceType
from ./samples/scripts/common/utility.sh:  

Remove multiple statements  "echo domainHomeSourceType is PV" 
from samples/scripts/common/utility.sh

If we want to print some INFO or ERROR message,  we must print the MESSAGE type.  Just by printing the message does not help to debug the issue.

See function printError() and function printInfo() in  samples/scripts/domain-lifecycle/helper.sh

File: kubernetes/samples/scripts/common/utility.sh (hunk 1 above)
Comment (bhavaniravichandran) [reply kubernetes/samples/scripts/common/utility.sh]: Removed

File: kubernetes/samples/scripts/common/utility.sh (hunk 2 above)
Comment (bhavaniravichandran) [reply kubernetes/samples/scripts/common/utility.sh]: Removed

**Comment by bhavaniravichandran** [comment]
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/4633/
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/4634/
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/4635/

**Review by anpanigr — APPROVED** [review]
lgtm

**Review by ddsharpe — APPROVED** [review]

**Review by mriccell — APPROVED** [review]

**Pull request merged**